judges/label-was-attached/label-was-attached.rb

```diff
@@ -34,6 +34,7 @@
   repeats 20
   over do |repository, issue|
     Fbe.octo.issue_timeline(repository, issue).each do |te|
+      Fbe.fb.query("(eq issue #{issue})").delete! if te[:status] == '404'
```

Comment:
@Suban05 all correct, but you will delete facts related to other repositories too. Better do this:

```
(and (eq where 'github') (eq repository #{repository}) (eq issue #{issue}))
```

Reply:
@yegor256 thanks! Also, I have added a test of this case